core/src/main/java/com/mapzen/android/graphics/MapzenMap.java

```diff
@@ -649,7 +643,8 @@ public void onMarkerPick(final MarkerPickResult markerPickResult, final float po
         mapView.post(new Runnable() {
           @Override public void run() {
             if (markerPickResult != null) {
-              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker()));
+              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker(),
```

Comment:
Ok this is a bit confusing to me, why create a new `BitmapMarker` in the listener interface here? I might not be fully understanding this here.

Looks like this was being done originally here: https://github.com/mapzen/android/pull/340/files#diff-acea21ee9a7821af52798402334ac2e1R638